tools/projmgr/templates/cdefault.yml

```diff
@@ -30,9 +28,9 @@ default:
         - -std=gnu11
       Link:
         - --specs=nano.specs
-        - --specs=rdimon.specs
         - -Wl,-Map=$elf()$.map
         - -Wl,--gc-sections
+        - -Wl,--no-warn-rwx-segments
```

Comment:
I think this change is dangerous as it would silence the warning that the application is open for a wide range of possible attacks.
Segments in an .elf file that requires both executable and write permission at the same time is vulnerable to a overflow attack and would allow to overwrite the application code in these segments to inject custom code into the application space and potentially leak sensitive information or take over the application (ransom attack for example).